src/MizarData/include/MizarData/SafeTimestampAdd.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef MIZAR_DATA_SAFE_TIMESTAMP_ADD_H_
+#define MIZAR_DATA_SAFE_TIMESTAMP_ADD_H_
+
+#include <stdint.h>
+
+#include <cstdint>
+#include <limits>
+
+[[nodiscard]] inline uint64_t SafeTimestampsAdd(const uint64_t a, const uint64_t b) {
```

Comment:
I think the name needs to expose what this function is actually doing. What about `NoWrappingAddition` or `AddNonWrapping` or something similar?